OpenXmlPowerTools/PtOpenXmlUtil.cs

```diff
@@ -5359,6 +5359,222 @@ public static class W14
+        public static XName lin = w15 + "lin";
+        public static XName lon = w15 + "lon";
+        public static XName lumMod = w15 + "lumMod";
+        public static XName lumOff = w15 + "lumOff";
+        public static XName miter = w15 + "miter";
+        public static XName noFill = w15 + "noFill";
+        public static XName numForm = w15 + "numForm";
+        public static XName numSpacing = w15 + "numSpacing";
+        public static XName nvContentPartPr = w15 + "nvContentPartPr";
+        public static XName paraId = w15 + "paraId";
+        public static XName path = w15 + "path";
+        public static XName pos = w15 + "pos";
+        public static XName props3d = w15 + "props3d";
+        public static XName prst = w15 + "prst";
+        public static XName prstDash = w15 + "prstDash";
+        public static XName prstMaterial = w15 + "prstMaterial";
+        public static XName r = w15 + "r";
+        public static XName rad = w15 + "rad";
+        public static XName reflection = w15 + "reflection";
+        public static XName rev = w15 + "rev";
+        public static XName rig = w15 + "rig";
+        public static XName rot = w15 + "rot";
+        public static XName round = w15 + "round";
+        public static XName sat = w15 + "sat";
+        public static XName satMod = w15 + "satMod";
+        public static XName satOff = w15 + "satOff";
+        public static XName scaled = w15 + "scaled";
+        public static XName scene3d = w15 + "scene3d";
+        public static XName schemeClr = w15 + "schemeClr";
+        public static XName shade = w15 + "shade";
+        public static XName shadow = w15 + "shadow";
+        public static XName solidFill = w15 + "solidFill";
+        public static XName srgbClr = w15 + "srgbClr";
+        public static XName stA = w15 + "stA";
+        public static XName stPos = w15 + "stPos";
+        public static XName styleSet = w15 + "styleSet";
+        public static XName stylisticSets = w15 + "stylisticSets";
+        public static XName sx = w15 + "sx";
+        public static XName sy = w15 + "sy";
+        public static XName t = w15 + "t";
+        public static XName textFill = w15 + "textFill";
+        public static XName textId = w15 + "textId";
+        public static XName textOutline = w15 + "textOutline";
+        public static XName tint = w15 + "tint";
+        public static XName uncheckedState = w15 + "uncheckedState";
+        public static XName val = w15 + "val";
+        public static XName w = w15 + "w";
+        public static XName wProps3d = w15 + "wProps3d";
+        public static XName wScene3d = w15 + "wScene3d";
+        public static XName wShadow = w15 + "wShadow";
+        public static XName wTextFill = w15 + "wTextFill";
+        public static XName wTextOutline = w15 + "wTextOutline";
+        public static XName xfrm = w15 + "xfrm";
+    }
+
+    public static class W16se
+    {
+        public static XNamespace w16se =
```

Comment:
As for the `W15` class, could you turn `public static` into `public static readonly`? In my PR, I have changed all other fields into `public static readonly` fields.